dhall-try/index.html

```diff
@@ -434,32 +461,88 @@ <h1 class="display-4">Tired of YAML?</h1>
 
     input.setValue(example0);
 
-    h0.setValue(`let input =
-      { relative = "daughter"
-      , movies   = [ "Boss Baby", "Frozen", "Moana" ]
-      }
+    h0.setValue(`-- ./curated-dependencies.dhall
+
+let dependencies = ./available-dependencies.dhall
+
+let universal =
+      [ dependencies.base
+      , dependencies.relude
+      ]
 
-let concatSep = https://prelude.dhall-lang.org/Text/concatSep
-
-    -- Dhall strips leading indentation for you
-in  ''
-    My \${input.relative} loves to watch \${concatSep ", " input.movies}
-    
-    How about you?
-    ''`);
-
-    h1.setValue(`{ -- Unlike YAML, Dhall does not accept YES|NO|ON|OFF
-  validDhallBools = [ True, False ]
-    , someNumbers = [ 1
-  ,
--- Dhall is not indentation-sensitive
-2, 3 ]
-  -- Field names that conflict with reserved identifiers must be quoted
-, \`True\` = True
-, version = "9.3"  {- Strings must be quoted
-
-                      All Dhall literals have unambiguous types -}
+let server =
+      [ dependencies.servant
+      , dependencies.servant-server
+      , dependencies.warp
+      ]
+
+let shell =
+      [ dependencies.foldl
+      , dependencies.turtle
+      , dependencies.optparse-applicative
+      , dependencies.optparse-generic
+      ]
+
+in  { universal = universal
+    , server = server
+    , shell = shell
+    }`);
+
+    d0.setValue(`$ dhall diff \\
+    'https://prelude.dhall-lang.org/v9.0.0/package.dhall' \\
+    'https://prelude.dhall-lang.org/v10.0.0/package.dhall'
+{ Natural = { + equal = …
+            , + greaterThan = …
+            , + greaterThanEqual = …
+            , + lessThan = …
+            , + lessThanEqual = …
+            , …
+            }
+, Text = { + default = …
+         , + defaultMap = …
+         , …
+         }
+, …
 }`);
+
+    h1.setValue(`$ cat ./Package.dhall 
```

Comment:
In the "Use a real programming language"-section I would have rather expected some use of functions or operators…

IMHO this example is also a bit Haskell-specific with the `Hackage` constructor…

Again, sorry for not offering something constructive

Reply:
@sjakobi: Whoops, replied to the wrong comment, so I'll rephrase my original comment here.

How about an example using assertions?  I think language support for tests are something that provides strong evidence that this is a real programming language and it also dovetails with the tweets/quote about insufficient testing of configuration changes.

Reply:
Yep, assertions are a great feature and would be nice to demonstrate here!